common/schema-builder/pom.xml

```diff
@@ -35,9 +35,8 @@
         
         <!-- Test -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
```

Comment:
Why is the scope not test anymore ?

Reply:
The scope is inherited from the managed dependency. I agree that it's clearer to still add the scope everywhere, but I've seen this in this project already, so I thought it would be more consistent.